src/rv32_template.c

```diff
@@ -155,8 +155,10 @@ RVOP(
     jal,
     {
         const uint32_t pc = PC;
+
```

Comment:
Don't do this. Minimize the necessary changes.
Indent with clang-format-18.

Reply:
> Don't do this. Minimize the necessary changes. Indent with clang-format-18.

clang-format-18 above will also work?

Reply:
I encountered incompatibility with clang-format-20.